.github/ISSUE_TEMPLATE/data_access_request.yml

```diff
@@ -0,0 +1,44 @@
+name: Data access request
+description: Request access to pseudopeople input data (beyond the sample included with pseudopeople)
+body:
+  - type: input
+    attributes:
+      label: "What is the name of your project?"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: "What is the purpose of your project?"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: "Who is involved in the project? Which of these people will have direct access to the pseudopeople input data?"
```

Comment:
We better get some sort of contact info, but maybe people don't want to enter it openly in github.  I have not thought this through!  @Ironholds : what do you recommend?

Reply:
Os has convinced me not to change the form for this.